src/vm/internal/compiler/lex.cc

```diff
@@ -841,10 +846,82 @@ static int skip_to(const char *token, const char *atoken) {
   }
 }
 
+void init_include_path() {
+  svalue_t *ret;
```

Comment:
Mine was the venerable Commodore 64.  I wrote my own 6510 assembler for it,
but never quite got macros working.  I did learn C on it as well, but I
forget the brand of compiler.

Back then, goto was considered useful, but easily abused.  That's the
attitude I wish we still had towards it, rather than the witch hunt
never-use-goto-or-you-will-be-flayed-alive mindset the object oriented
crowd brought to the club.

On Mon, Jan 18, 2016 at 5:54 AM, Shea690901 <notifications@github.com>
wrote:

> In src/vm/internal/compiler/lex.cc
> <https://github.com/fluffos/fluffos/pull/313#discussion_r49984678>:
>
> > @@ -841,10 +846,82 @@ static int skip_to(const char *token, const char *atoken) {
> >    }
> >  }
> >
> > +void init_include_path() {
> > +  svalue_t *ret;
>
> Probably because there's much to much ckde out there made unreadable by
> too many goto's, sometimes even leaving one function jumping into the
> middle of another *shudder*.
> And concerning your age:
> I teached myself programming on an "Amstrad CPC464" under "Locomotive
> BASIC", later Z80-Assembler and finaly K&R C (the last one under CP/M) on
> the same 'machine' ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fluffos/fluffos/pull/313/files#r49984678>.
>